PlaybackLab/ControlsLockScreenSampleApp/ControlsLockScreenSampleApp/Models/PlayerSelectionOption.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  PlayerSelectionOption.swift
+//  ControlsLockScreenSampleApp
+//
+//  Created by Carlos Ceja on 10/18/17.
+//  Copyright © 2017 Ooyala. All rights reserved.
+//
+
+import Foundation
+
+class PlayerSelectionOption: NSObject {
+  
+  private var domain: OOPlayerDomain
```

Comment:
Make all these properties as read-only. For example: `public private(set) var domain: OOPlayerDomain`

Reply:
Done.